guides/source/active_storage_overview.md

```diff
@@ -531,68 +531,102 @@ It's important to know that the file is not yet available in the `after_create`
 Analyzing Files
 ---------------
 
-Active Storage [analyzes](https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyze) files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
+Active Storage [analyzes][] files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
```

Comment:
I would leave this as-is for the reasons mentioned in https://github.com/rails/rails/pull/40712#discussion_r534314371.